src/sentry/api/endpoints/organization_region.py

```diff
@@ -65,12 +64,7 @@ def convert_args(
 
         try:
             # We don't use the lookup since OrganizationMapping uses a BigIntField for organization_id instead of a ForeignKey
-            if (
-                id_or_slug_path_params_enabled(
-                    self.convert_args.__qualname__, str(organization_id_or_slug)
-                )
-                and str(organization_id_or_slug).isdecimal()
-            ):
+            if str(organization_id_or_slug).isdecimal():
                 org_mapping = OrganizationMapping.objects.get(
```

Comment:
Could you remind me again why we don't use the lookup for `OrganizationMapping`

Reply:
OrganizationMapping is a little different where it doesn't identify an organization by id, rather `organization_id`. It is a 1-off scenario so instead of making another lookup, we just added the logic in the endpoint.